### PR TITLE
Add typescript typedefs for the main components

### LINKED ISSUE #10: Add typescript typings
Would it be possible to write typescript typings and add them to DefinitelyTyped? I'm not yet confident enough with typescript to do it myself.

### PR DESCRIPTION
**Pull request by Seldaek**
We are switching to TypeScript at the moment so I needed this :)

Fixes #10

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Context Menu implemented in React",
   "main": "modules/index.js",
   "module": "es6/index.js",
+  "typings": "src/index.d.js",
```
Comment (vkbansal) [inline package.json]: shouldn't this be `.ts`?

Comment (Seldaek) [inline package.json]: Yes it should of course, fixed :)

**Comment by Seldaek** [comment]
Cool thanks, if you can push out a new release as well that'd be great :)

**Comment by vkbansal** [comment]
done

**Pull request merged**